README.md

```diff
@@ -4,34 +4,42 @@ This library provides support classes for creating and processing
 Decentralized Identifiers for Veres One. This library enables a developer to:
 
 * Create a Veres One DID
-* Perform Veres One key rotation
 * Generate Veres One cryptographic proofs
 
-## The Promises API
-
-  * api.generate(options)
-
-## The Callback API (node.js)
-
-  * api.generate(options, (err, didDocument))
-
-## Quick Examples
-
 ```
 npm install did-veres-one
 ```
 
 ```js
-const didv1 = require('did-veres-one');
-const options = {};
+const v1 = require('did-veres-one');
+
+// See Configuration below for list of options
+const options = {mode: 'dev', httpsAgent, hostname: 'localhost:12345'};
+const veresDriver = v1.driver(options);
 
-// generate the DID document
-const didDocument = await didv1.generate();
+// this can now be used with did-io, or directly
```

Comment:
I don't find this comment helpful.  I may not now what `did-io` is, and I do not know what `this` can now be used.

Maybe just strike it for now?

Reply:
Good point, removed.